spring-cloud-stream-binders/spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitProducerProperties.java

```diff
@@ -108,6 +111,7 @@ public void setBatchSize(int batchSize) {
 		this.batchSize = batchSize;
 	}
 
+	@Min(value = 1, message = "Batch Buffer Limit should be greater than or equal to 1")
```

Comment:
I am not sure a buffer limit of 1 makes much sense; maybe all these should just be "must be greater than zero".